idaes/gas_solid_contactors/unit_models/bubbling_fluidized_bed.py

```diff
@@ -1165,20 +1301,26 @@ def bubble_heat_transfer(b, t, x):
                              self.length_domain,
                              doc="Gas Emulsion - Heat Transfer")
             def gas_emulsion_heat_transfer(b, t, x):
-                return (b.gas_emulsion.heat[t, x] ==
+                return (b.gas_emulsion.heat[t, x]*pyunits.m**3 ==
```

Comment:
1. Why are you multiplying by `m**3` here? This implies that units are not consistent in this constraint.
2. Hardcoding `pyunits.m` here presumes that length is defined in metres - you should always go via the units meta data to get units.

Reply:
In regards to this and other related comments below, I re-checked and corrected units for the heat and mass transfer variables. This removed some of the floating unit multipliers, and I attached remaining ones to constants wrapped in `pyunit.convert` statements to use the metadata units.